documentation/modules/cruise-control/proc-implementing-rebalance-proposal.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// assembly-cruise-control-concepts.adoc
+
+[id='proc-implementing-optimization-proposal-{context}']
+
+= Implementing an optimization proposal
+
+You can implement an optimization proposal with a status of `ProposalReady`.
+Cruise Control will then reassign partitions to brokers as summarized in the optimization proposal.
+
+WARNING: This is not a dry run. 
+Review the optimization proposal before you implement it.
+
+.Prerequisites
+
+* You have xref:proc-generating-optimization-proposals-{context}[generated an optimization proposal].
+
+* The optimization proposal is ready to implement (the `KafkaRebalance` custom resource status is `ProposalReady`).
+
+.Procedure
+
+Perform these steps for the optimization proposal that you want to implement:
+
+. If the `KafkaRebalance` status has been `ProposalReady` for some time, refresh the optimization proposal in case it has become out of date:
```

Comment:
We should add a prominent **NOTE:** to this with some more explanation:

**NOTE**: If the `KafkaRebalance` status has been in the `ProposalReady` state for some time, then that proposal may no longer be applicable to the current state of the Kafka Cluster. In this case you can refresh the optimization proposal to make sure it uses the latest cluster metrics.

Reply:
Good idea.